### PR TITLE
new image for the og tags & better display when sharing.

### PR DESCRIPTION
**Pull request by luxumbra**
- Also added a few tweaks for 2k displays

## Overview
A small update which adds an updated image for the og tags and sorts some minor visual issues on 2k displays.

### CODE REVIEW

File: packages/web/components/Landing/LandingHeader.tsx [hunk 1]
```diff
@@ -83,24 +83,26 @@ export const LandingHeader: React.FC = () => {
               position: 'relative',
               flexDirection: 'column',
               justifyContent: 'space-around',
-              width: ['2.5rem'],
-              height: ['2.5rem'],
+              width: { base: '2.25rem', xl: '2.5rem', '4xl': '3.1rem' },
+              height: { base: '2.25rem', xl: '2.5rem', '4xl': '3.1rem' },
+              borderRadius: { base: '5px', xl: '15px', '4xl': '20px' },
```
Comment (dysbulic) [inline packages/web/components/Landing/LandingHeader.tsx:88]:
```suggestion
              borderRadius: { base: '0.25rem', xl: '1rem', '4xl': '1.25rem' },
```

File: packages/web/components/Landing/LandingHeader.tsx [hunk 2]
```diff
@@ -83,24 +83,26 @@ export const LandingHeader: React.FC = () => {
...
               background: 'transparent',
               border: 'none',
               cursor: 'pointer',
               padding: 0,
               marginRight: 0,
               zIndex: 401,
-              overflow: 'hidden',
+              // overflow: 'clip',
```
Comment (dysbulic) [inline packages/web/components/Landing/LandingHeader.tsx:95]:
```suggestion
```

File: packages/web/components/Landing/LandingHeader.tsx [hunk 3]
```diff
@@ -83,24 +83,26 @@ export const LandingHeader: React.FC = () => {
...
                 outline: 'none',
                 background: 'transparent',
                 boxShadow: 'none',
               },
               div: {
-                width: ['2.5rem'],
-                // height: ['0.08rem', '0.1rem', '0.2rem'],
-                borderRadius: ['5px', '5px', '10px'],
+                width: { base: '100%' },
+                height: { base: '100%' },
```
Comment (dysbulic) [inline packages/web/components/Landing/LandingHeader.tsx:103]:
```suggestion
                w: "full",
                h: "full",
```

File: packages/web/components/Landing/LandingHeader.tsx [hunk 4]
```diff
@@ -83,24 +83,26 @@ export const LandingHeader: React.FC = () => {
...
+                borderRadius: { base: '5px', xl: '15px', '4xl': '20px' },
```
Comment (dysbulic) [inline packages/web/components/Landing/LandingHeader.tsx:104]:
```suggestion
                borderRadius: { base: '0.25rem', xl: '1rem', '4xl': '1.25rem' },
```

File: packages/web/components/Landing/LandingHeader.tsx [hunk 5]
```diff
@@ -83,24 +83,26 @@ export const LandingHeader: React.FC = () => {
...
+                // overflow: 'clip',
```
Comment (dysbulic) [inline packages/web/components/Landing/LandingHeader.tsx:105]:
```suggestion
```

File: packages/web/components/Landing/LandingHeader.tsx [hunk 6]
```diff
@@ -260,39 +262,14 @@ const NavLink = ({
...
-    </Box>
-  </Box>
-);
-
 export const MenuIcon2SVG = ({ toggle }: { toggle: boolean }) => (
   <Box>
     <Box
       as="svg"
-      width={['2.5rem']}
+      width={{ base: '2.25rem', xl: '2.5rem', '4xl': '2.9em' }}
```
Comment (dysbulic) [inline packages/web/components/Landing/LandingHeader.tsx:269]:
```suggestion
      w={{ base: '2.25rem', xl: '2.5rem', '4xl': '2.9rem' }}
```

File: packages/web/components/Landing/LandingHeader.tsx [hunk 7]
```diff
@@ -260,39 +262,14 @@ const NavLink = ({
...
+      position="absolute"
+      ml="2px"
+      mt="2px"
```
Comment (dysbulic) [inline packages/web/components/Landing/LandingHeader.tsx:272]:
```suggestion
      ml={0.5}
      mt={0.5}
```

File: packages/web/components/Landing/LandingHeader.tsx [hunk 8]
```diff
@@ -305,7 +282,7 @@ export const MenuIcon2SVG = ({ toggle }: { toggle: boolean }) => (
         d="M46.8937 23.64C46.8937 36.4827 36.4827 46.8937 23.64 46.8937C10.7973 46.8937 0.386262 36.4827 0.386262 23.64C0.386262 10.7973 10.7973 0.386262 23.64 0.386262C36.4827 0.386262 46.8937 10.7973 46.8937 23.64Z"
         stroke="white"
         strokeOpacity="0.9"
-        strokeWidth="0.772524"
+        strokeWidth="1"
```
Comment (dysbulic) [inline packages/web/components/Landing/LandingHeader.tsx:285]:
```suggestion
        strokeWidth={1}
```

File: packages/web/pages/index.tsx [hunk 9]
```diff
@@ -99,9 +103,9 @@ const Landing: React.FC = () => {
     <>
       <HeadComponent
         title="MetaGame: A Massive Online Coordination Game!"
-        description="To find your metagame means to play life in the optimal way. Coordinating with others on building a better world; doing things that create a a positive impact make you happy AND earn you money."
+        description="To find your metagame means to play life in the optimal way. Coordinating with others to build a better world; make a positive impact, make you happy AND earn you money."
```
Comment (dysbulic) [inline packages/web/pages/index.tsx:106]:
```suggestion
        description="To find your metagame means to play life in the optimal way. Coordinating with others to build a better world; make a positive impact, make you happy, 𝘈𝘕𝘋 earn you money."
```

File: packages/web/pages/index.tsx [hunk 10]
```diff
@@ -99,9 +103,9 @@ const Landing: React.FC = () => {
...
         url="https://metagame.wtf"
```
Comment (dysbulic) [inline packages/web/pages/index.tsx:107]:
```suggestion
        url={hostName}
```

**Review by dysbulic — COMMENTED** [review]
Switched some pixels to `rem`s & shortened some `width`s to `w`s.

Still waiting on the test instance to come up to actually look at it.

**Comment by luxumbra** [comment]
Thank you for the speedy review @dysbulic. 🙏🏻 

I have addressed all of your feedback and have also tweaked the way I am getting the site origin.

File: packages/web/pages/index.tsx (hunk 9 above)
Comment (dysbulic) [reply packages/web/pages/index.tsx]: Don't like the Oxford comma?

Comment (luxumbra) [reply packages/web/pages/index.tsx]: Its not that, I had already updated the text to use `&amp;` in an effort to reduce the char count from 190 to the recommended limit of 160 but forgot to add in the oxford comma. 🤦🏻

Comment (dysbulic) [reply packages/web/pages/index.tsx]: ✅

**Comment by dysbulic** [comment]
Since it looks like you'll have to push at least one more change, `url={`${hostName.current}`}` should be `url={hostName.current}`.

**Comment by luxumbra** [comment]
> Since it looks like you'll have to push at least one more change, `` url={`${hostName.current}`} `` should be `url={hostName.current}`.

Hmmm, yeah I tried that but got an error on the url property which went away when i used a literal. Not sure why the build is now failing though.

**Comment by luxumbra** [comment]
> Since it looks like you'll have to push at least one more change, `` url={`${hostName.current}`} `` should be `url={hostName.current}`.

and should i perhaps rebase my last few commits so as not to milk the old XP?

**Comment by luxumbra** [comment]
Marking as draft whilst I get sign off for the image from designers. Was an afterthought. Apologies for the to-ing & fro-ing.

**Review by alalonde — APPROVED** [review]

**Comment by luxumbra** [comment]
Thanks @alalonde 🙏🏻

**Pull request merged**